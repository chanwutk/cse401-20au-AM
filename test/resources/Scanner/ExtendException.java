class A extends RuntimeException {}
class B extends A {}
class C {}
class D extends C {}