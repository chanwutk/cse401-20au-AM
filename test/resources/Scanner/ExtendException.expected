CLASS ID(A) EXTENDS RUNTIME_EXCEPTION LBRACKET RBRACKET CLASS ID(B) EXTENDS ID(A) LBRACKET RBRACKET CLASS ID(C) LBRACKET RBRACKET CLASS ID(D) EXTENDS ID(C) LBRACKET RBRACKET